double/double.mbt

```diff
@@ -274,3 +274,27 @@ pub fn to_string(self : Double) -> String {
 pub impl Show for Double with output(self, logger) {
   logger.write_string(self.to_string())
 }
+
+fn max(a : Double, b : Double) -> Double {
+  if a > b {
+    a
+  } else {
+    b
+  }
+}
+
+pub fn is_close(
+  self : Double,
+  other : Double,
+  ~relative_tolerance : Double = 1.0e-09,
+  ~absolute_tolerance : Double = 0.0
+) -> Bool {
```

Comment:
add sanity check on the inputs like 

https://github.com/python/cpython/blob/d683f49a7b0635a26150cfbb398a3d93b227a74e/Modules/mathmodule.c#L3115